frontend/src/concepts/analyticsTracking/useTrackUser.ts

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useUser } from '~/redux/selectors';
+import { useAccessReview } from '~/api';
+import { AccessReviewResourceAttributes } from '~/k8sTypes';
+import { IdentifyEventProperties } from '~/concepts/analyticsTracking/trackingProperties';
+
+export const useTrackUser = (username?: string): IdentifyEventProperties => {
+  const { isAdmin } = useUser();
+  const [anonymousId, setAnonymousId] = React.useState<string | undefined>(undefined);
+
+  const createReviewResource: AccessReviewResourceAttributes = {
+    group: 'project.openshift.io',
+    resource: 'projectrequests',
+    verb: 'create',
+  };
+  const [allowCreate] = useAccessReview(createReviewResource);
```

Comment:
This makes a network request but the value isn't used in the current implementation.

Reply:
It is used in the returned props.

Reply:
It may be returned but I do not see it being consumed by the current implementation.

Reply:
It is used inside the `fireIdentifyEvent() `and passed to segment as user properties.

Reply:
`fireIdentifyEvent` is only using `properties.anonymousID` but no one uses `canCreateProjects` (which comes from `allowCreate` in this function)